### ISSUE TITLE
Snakes and Ladders - Language, History and Indo-China

### ISSUE THREAD

**Issue by vaibhavswire** [vaibhavswire/publicbookshelf#2]
Abstract - The universal political jingle of language narratives, history sounds and  the onset of another Eastern trial by stagnation.

Like with historians, so with Chinese history. While you are learning there is so much happening and once you cross the main benchmarks of learning history like Chinese history you do not know where to go now. Not wonderment. Not exasperation. Plain simple dumbfoundedness, may arrive.

Chinese diplomats narrate that their officials annnals can be read to see what was happening two thousand years ago, in the same language. It is a near true and accurate statement.

And what is Sino languages? Hmong-Mien and Krai-Dai are not it, they were later developed like American English. Was Sino-Tibetan? Old Chinese has no tones. Middle Chinese is read from a Rime table book and has consonant clusters at the end of the syllables. Sino languages are thus in debate much like the Nostratic desires of Latin script dictionaries. The Novogorodsky Yakut Cyrillic is waiting to pounce. 

Cantonese is the pure bred Southerner cowboy. And much like Terry Pratchett Brahmins of India 800 years of history is in a debate of one grammarian question - is language only sound, or intuition?

To summarize the history of Sanskrit literature since the 13th century (After Abhinavgupta) there are only four books  -  the commentaries on Kashika, a commentary on Panini’s Ashtadhyayi, a grammar book. The commentaries such as Helaraja on Bhrtrhari’s Vakyapadiya another grammar book. Seek a good bibiliography of indian Paleography and Manuscripts. Lastly follow the government of India’s manuscript publishing at namami.org - there is nothing more to it. 

“ ‘We have animals up on the mountain and if you do not milk them then the animals will suffer’ - Ikram Sevim “ 
Mark Z. Danielewski,  The Familiar, Volume III

References -

I Ching for the positivists and Confucianism for the negativistic historical approach.
